google/resource-snippets/logging-v2/test_alternatives.sh

```diff
@@ -0,0 +1,39 @@
+set -e
+
+# assumes that the following projects are created and kubectl context is set to a cluster in KRM_PROJECT
+TF_PROJECT_ID=[TF_PROJECT_ID]
+DM_PROJECT_ID=[DM_PROJECT_ID]
+
+gcloud auth application-default login
+
+# Create DM resources
+gcloud config set project $DM_PROJECT_ID
+gcloud services enable deploymentmanager.googleapis.com --project $DM_PROJECT_ID
+gcloud services enable cloudkms.googleapis.com --project $DM_PROJECT_ID
```

Comment:
This should be logging.googleapis.com. Same with line 18

Reply:
updated